base.js

```diff
@@ -1,4 +1 @@
-'use strict'
-var core = require('./core'),
-    merge = core.util.objectMerge
-module.exports = merge(merge({}, core), require('./transducers'))
+module.exports = require('./lib/base')
```

Comment:
If you wanted, I believe `export * from './lib/base'` would have the same effect